src/ontology/bspo-edit.owl

```diff
@@ -991,6 +996,69 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/BSPO_0020002> "ha
 ObjectPropertyDomain(<http://purl.obolibrary.org/obo/BSPO_0020002> <http://purl.obolibrary.org/obo/BSPO_0000010>)
 ObjectPropertyRange(<http://purl.obolibrary.org/obo/BSPO_0020002> <http://purl.obolibrary.org/obo/CARO_0000000>)
 
+# Object Property: <http://purl.obolibrary.org/obo/BSPO_0030003> (dorsalmost part of)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://orcid.org/0000-0003-2699-3066") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/BSPO_0030003> "x dorsalmost_part_of y and x part_of y 
+no z more dorsal than x
+x is adjacent to the dorsal boundary of y")
```

Comment:
I haven't written too many of these relation definitions, but here is a stab at it.

```suggestion
AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://orcid.org/0000-0003-2699-3066") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/BSPO_0030003> "x dorsalmost_part_of y iff x part_of y and there is no z which is part_of y and is more dorsal than x but not part_of x; x is adjacent to the dorsal boundary of y")
```

Reply:
If that sounds good to you, you could update the others to match.

Reply:
What does "iff" mean?

Reply:
"If and only if"